workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -7,26 +7,44 @@ public abstract class MobileWorkerConnector : WorkerConnector
     {
         protected abstract string GetHostIp();
 
+        protected override ConnectionParameters GetConnectionParameters(string workerType, ConnectionService service)
+        {
+            return new ConnectionParameters
+            {
+                WorkerType = workerType,
+                Network =
+                {
+                    ConnectionType = NetworkConnectionType.Tcp,
+                    UseExternalIp = true,
+                },
+                EnableProtocolLoggingAtStartup = false,
+                DefaultComponentVtable = new ComponentVtable(),
```

Comment:
This feels weird to be in the worker connector classes. I think it needs a larger refactor to get it out so there is no change to make for now, but it does feel weird.

Reply:
I don't find it too weird, as the worker connector takes care of specifying the exact connection logic, but I agree that we could probably separate it out a bit more. Gonna create a ticket to redesign this in the near future